README.md

```diff
@@ -33,6 +33,8 @@ KubeEdge is composed of the following components:
 
 - **Edged:** an agent that runs on edge nodes and manages containerized applications.
 - **EdgeHub:** a web socket client responsible for interacting with Cloud Service for the edge computing (like Edge Controller as in the KubeEdge Architecture). This includes syncing cloud-side resource updates to the edge, and reporting edge-side host and device status changes to the cloud.
+- **CloudHub:** a web socket server responsible for watching changes at the clouds side, cache and sending messages to EdgeHub.
+- **EdgeController:** an extended kubernetes controller which manages edge nodes and pods metadata so that the data can target to a specific edge node. 
```

Comment:
so that the data can target to a specific edge node -> so that the data can be targeted to a specific edge node.

Reply:
Done, Thanks